### PR TITLE
twist_controller -> picknik_twist_controller

### PR DESCRIPTION
**Pull request by moriarty**
1. prefix twist_controller with picknik_twist_controller

When we merge twist_controller into ros2_controllers we can depricate this one and not have naming conflicts as users migrate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ We plan to land this upstream in ros2_controllers: https://github.com/ros-contro
 ### Usage
 Example of instance within `ros2_control` can be found in the [configuration file](https://github.com/PickNikRobotics/ros2_kortex/blob/main/kortex_description/arms/gen3/7dof/config/ros2_controllers.yaml).
 Defined interfaces have to exist within the `hardware_interface::SystemInterface` implemented for the robot (e.g. driver).
-Stop any commanding controller and then start `TwistController`to use kinova's builtin cartesian twist controller.
+Stop any commanding controller and then start `PicknikTwistController`to use kinova's builtin cartesian twist controller.
```
Comment (MarqRazz) [inline README.md:17]: Can we remove `Kinova`?

File: picknik_twist_controller/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.8)
```
Comment (MarqRazz) [inline picknik_twist_controller/CMakeLists.txt:1]: Should we update the version on this?

File: picknik_twist_controller/controller_plugins.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<library path="picknik_twist_controler">
+  <class name="picknik_twist_controler/PicknikTwistControler" type="picknik_twist_controler::PicknikTwistControler" base_class_type="controller_interface::ControllerInterface">
+  <description>
+    6dof twist controller
```
Comment (MarqRazz) [inline picknik_twist_controller/controller_plugins.xml:4]: Can we update this to say it actually does? Subscribes to a twist message and forwards it to the hardware.

File: picknik_twist_controller/package.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
-  <name>twist_controller</name>
+  <name>picknik_twist_controller</name>
   <version>0.0.0</version>
   <description>TODO: Package description</description>
```
Comment (MarqRazz) [inline picknik_twist_controller/package.xml:6]: Update description.

File: README.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ We plan to land this upstream in ros2_controllers: https://github.com/ros-contro
 ### Usage
 Example of instance within `ros2_control` can be found in the [configuration file](https://github.com/PickNikRobotics/ros2_kortex/blob/main/kortex_description/arms/gen3/7dof/config/ros2_controllers.yaml).
 Defined interfaces have to exist within the `hardware_interface::SystemInterface` implemented for the robot (e.g. driver).
-Stop any commanding controller and then start `TwistController`to use kinova's builtin cartesian twist controller.
+Stop any commanding controller and then start `PicknikTwistController`to forward twist msgs to the hardwares builtin cartesian twist controller.
```
Comment (abake48) [inline README.md:17]: No need to block on this but I have 1 small nitpick. Everything else looks good to me.
```suggestion
Stop any commanding controller and then start `PicknikTwistController`to forward twist msgs to the hardware's built-in cartesian twist controller.
```

**Review by abake48 — APPROVED** [review]

**Pull request merged**